Libs/DICOM/Widgets/ctkDICOMAppWidget.cpp

```diff
@@ -273,8 +277,10 @@ ctkDICOMAppWidget::ctkDICOMAppWidget(QWidget* _parent):Superclass(_parent),
   d->ToolBar->setToolButtonStyle(Qt::ToolButtonTextUnderIcon);
 
   //Initialize Q/R widget
+#if QT_VERSION >= QT_VERSION_CHECK(5,0,0)
```

Comment:
The check should be done based on enabled/disabled features. We should avoid features disappearing silently based on detected Qt version.